mods/disable-navigation-bar.wh.cpp

```diff
@@ -0,0 +1,62 @@
+// ==WindhawkMod==
+// @id              disable-navigation-bar
+// @name            Disable Navigation Bar
+// @description     Disables the navigation bar in file explorer
+// @version         1.0.0
+// @author          ItsProfessional
+// @github          https://github.com/ItsProfessional
+// @include         *
```

Comment:
How about adding the same note as with the previous mod regarding targeting all apps?

Reply:
Oh I forgot about that